src/app/shared/change-password-dialog/change-password-dialog.component.html

```diff
@@ -0,0 +1,34 @@
+<h1 mat-dialog-title>Change Password</h1>
+<div mat-dialog-content>
+  
+  <form [formGroup]="changePasswordForm">
+    <div fxLayout="row wrap" fxLayout.lt-md="column" fxLayoutGap="2%">
+      <mat-form-field fxFlex="96%">
+        <mat-label>Password</mat-label>
+        <input matInput [(ngModel)]="password" formControlName="password" type="password">
```

Comment:
We aren't working on `[(ngModel)]` approach for handling values of form fields. Please take look on other components as how they are implemented.